pio-system: add page (#5523)
tldr-pages/tldr: pages/common/pio-system.md

# pio system

> Miscellaneous system commands for PlatformIO.
> More information: <https://docs.platformio.org/en/latest/core/userguide/system/index.html>.

> - Install shell completion for the current shell (supports fish, bash, zsh and powershell):

Reviewer: Alphabetical (with Windows at the end):
```suggestion
- Install shell completion for the current shell (supports bash, fish, zsh and powershell):
```

`pio system completion install`

- Uninstall shell completion for the current shell:

`pio system completion uninstall`

> - Print system-wide PlatformIO information:

Reviewer 2: ```suggestion
- Display system-wide PlatformIO information:
```

`pio system info`

- Remove unused PlatformIO data:

`pio system prune`

- Remove only cached data:

`pio system prune --cache`

> - Show unused PlatformIO data to be removed without removing it:

Reviewer: ```suggestion
- List unused PlatformIO data that would be removed but do not actually remove it:
```
